cmd/scanner/scanner.go

```diff
@@ -71,46 +91,60 @@ func getCloudProviderAccounts() ([]inventory.Account, error) {
 			account.Key("user").String(),
 			account.Key("key").String(),
 		)
-		accounts = append(accounts, *newAccount)
+		if err := s.inventory.AddAccount(*newAccount); err != nil {
+			return err
+		}
 	}
 
-	return accounts, nil
+	return nil
 }
 
 // createStockers creates and configures stocker instances for each provided account to be inventoried.
-func createStockers(accounts []inventory.Account) error {
-	for _, account := range accounts {
+func (s *Scanner) createStockers() error {
+	for i := range s.inventory.Accounts {
+		account := s.inventory.Accounts[i]
 		switch account.Provider {
 		case inventory.AWSProvider:
-			logger.Info("Adding the AWS account to be inventoried", zap.String("account", account.Name))
-			stockers = append(stockers, stocker.NewAWSStocker(account))
+			s.logger.Info("Adding the AWS account to be inventoried", zap.String("account", account.Name))
+			s.stockers = append(s.stockers, stocker.NewAWSStocker(&account, logger))
 		case inventory.GCPProvider:
-			logger.Warn("Failed to scan GCP account",
+			logger.Warn("Can't scan GCP account",
```

Comment:
I think we should be consistent in the choice of terms when logging
`can't` used here, `Failed to` in `internal/stocker/aws.go`
Personally I prefer `Failed to ....`

Reply:
Sounds fair. Done